config/rbac/role.yaml

```diff
@@ -63,4 +68,11 @@ rules:
   - get
   - patch
   - update
-
+- apiGroups:
```

Comment:
Notification Manager does not need the authority of `rolebindings`. [This](https://github.com/kubesphere/notification-manager/blob/master/controllers/notificationmanager_controller.go#L68) should be deleted.

Reply:
Will do, this marker was in place before the PR.